### PR TITLE
37 - Migrate from laravel-mix to vue-cli

### PR DESCRIPTION
**Pull request by fbrissi**
#37 

I'm not sure if it's correctly.  
And command `vue-cli-service lint` I did not find how to use `eslintrc.prod.json` file

### CODE REVIEW

**Comment by fbrissi** [comment]
Rigth, send it to me please.

File: .editorconfig [hunk 1]
```diff
@@ -9,5 +9,4 @@ trim_trailing_whitespace = true
 charset = utf-8
 
 [{*.json,.*rc,*.yml}]
-indent_style = space
```
Comment (gabrielboliveira) [inline .editorconfig]: Please don't remove this

Comment (fbrissi) [reply .editorconfig]: But, doesn't redundante in `[*]`?

Comment (gabrielboliveira) [reply .editorconfig]: Thats a good point. Didn't think of that. Looks like its really redundant

**Pull request merged**